<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePasswordDialog</class>
 <widget class="QDialog" name="ChangePasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>修改密码</string>
  </property>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>原密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>新密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确认密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>92</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>确认修改</string>
   </property>
  </widget>
  <widget class="QLabel" name="report">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
